vhdbuilder/packer/install-dependencies.sh

```diff
@@ -199,10 +206,19 @@ for p in ${packages[*]}; do
   #getting metadata for each package
   name=$(echo "${p}" | jq .name -r)
   PACKAGE_VERSIONS=()
-  returnPackageVersions ${p} ${OS} ${OS_VERSION}
+  if [[ "${OS}" == "${MARINER_OS_NAME}" && "${isKata}" == "true" ]]; then
+    os=${MARINER_KATA_OS_NAME}
+  fi
+  returnPackageVersions ${p} ${os} ${OS_VERSION}
   PACKAGE_DOWNLOAD_URL=""
-  returnPackageDownloadURL ${p} ${OS} ${OS_VERSION}
+  returnPackageDownloadURL ${p} ${os} ${OS_VERSION}
   echo "In components.json, processing components.packages \"${name}\" \"${PACKAGE_VERSIONS[@]}\" \"${PACKAGE_DOWNLOAD_URL}\""
+  
+  # if ${PACKAGE_VERSIONS[@]} count is 0, skip to next package
+  if [[ ${#PACKAGE_VERSIONS[@]} -eq 0 ]]; then
```

Comment:
now that we're using `<SKIP>`, would this case ever happen?

Reply:
I left a fool-proof mechanism here in case someone leave the array empty or undefined, although in the readme we ask people to put `<SKIP>`. What do you think?

Reply:
yep sounds good, just wanted to clarify